app/code/Magento/ImportExport/view/adminhtml/templates/export/form/before.phtml

```diff
@@ -89,7 +89,7 @@ require([
             form.action   = oldAction;
         } else {
             alert({
-                content: 'Invalid data'
+                content: '<?= $block->escapeHtml(__('Invalid data')); ?>'
```

Comment:
Please, use `mage/translation` instead of block method

Reply:
I don't know the reason but the first time of this PR, I use the mage/translate. And it can not translate. Could you please try it?

![image](https://user-images.githubusercontent.com/8234209/63036620-93542980-bee7-11e9-8da2-c2ee406802c8.png)

Reply:
@sidolov